kie-soup-dataset/kie-soup-dataset-elasticsearch/pom.xml

```diff
@@ -224,13 +224,14 @@
       <groupId>com.spotify</groupId>
       <artifactId>docker-client</artifactId>
       <scope>test</scope>
-      <exclusions>
-        <exclusion>
-          <groupId>org.glassfish.hk2.external</groupId>
-          <artifactId>javax.inject</artifactId>
-        </exclusion>
-      </exclusions>
     </dependency>
+
+    <dependency>
+      <groupId>org.glassfish.jersey.inject</groupId>
+      <artifactId>jersey-hk2</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
if this relates to removed exclusion above is, does it here included for any reason?

Reply:
The new Spring version management broke some tests: `Caused by: java.lang.ClassNotFoundException: org.glassfish.hk2.api.MultiException`. This [Github issue](
https://github.com/tdomzal/junit-docker-rule/issues/42) describes the same problem I had with the broken tests, and adding this dependency fixed the problem. This is not related to the removed exclusion above AFAIK (which wasn't removed but rather moved to the `kie-soup-parent`)